src/type/scalars.ts

```diff
@@ -14,8 +14,8 @@ import { GraphQLScalarType } from './definition';
 //
 // n.b. JavaScript's integers are safe between -(2^53 - 1) and 2^53 - 1 because
 // they are internally represented as IEEE 754 doubles.
-const MAX_INT = 2147483647;
-const MIN_INT = -2147483648;
+export const MAX_INT = 2147483647;
```

Comment:
Please convert above comment to JSDoc
If you have time please write a separate docs for both min and max

Reply:
Also since we making it public please rename it to `GRAPHQL_INT_MAX` and `GRAPHQL_INT_MIN`.

Reply:
👍🏻